src/relayFeeCalculator/chain-queries/queries.e2e.ts

```diff
@@ -2,6 +2,8 @@
 // NODE_URL_42161
 // NODE_URL_288
 // NODE_URL_10
+// NODE_URL_1
+// NODE_URL_137
```

Comment:
These two environment variables may need to be added to the e2e test file.

Reply:
[That seems easy to do](https://docs.github.com/en/actions/learn-github-actions/environment-variables)

Alternatively we could use the public [infura key](https://github.com/UMAprotocol/protocol/blob/master/packages/common/src/ProviderUtils.ts#L38) here but I do think setting env vars in github actions is more sustainable

Reply:
Do you want to try to add these to the github actions file in `./.github/`?